x-pack/plugins/osquery/server/routes/fleet_wrapper/get_osquery_available.ts

```diff
@@ -0,0 +1,47 @@
+/*
```

Comment:
Do you happen to have a functional test for this api?

Reply:
I removed this API route.